packages/design-picker/src/__tests__/integration.test.tsx

```diff
@@ -45,4 +46,18 @@ describe( '<DesignPicker /> integration', () => {
 			)
 		);
 	} );
+
+	( [ 'light', 'dark' ] as Props[ 'theme' ][] ).forEach( ( theme ) =>
```

Comment:
Should we open a new issue to track improving these integration tests (ie. trying to check computed styles for borders and text colors)?

cc @StefanNieuwenhuis and @razvanpapadopol

Reply:
I've opened https://github.com/Automattic/wp-calypso/issues/52370 to track this.

- created also a new Milestone to keep track of DesignPicker stability issues like this:
https://github.com/Automattic/wp-calypso/milestone/354
- assigned https://github.com/Automattic/wp-calypso/issues/51743 to the new milestone.

cc: @Automattic/luna

Reply:
Awesome, thank you!